runtime/manifest-parser.peg

```diff
@@ -858,6 +858,8 @@ TopLevelIdent
     expected(`a top level identifier (e.g. "${fixed}")`);
   }
 
+TopLevelIdentBothCases = upperIdent / lowerIdent { return text(); }
```

Comment:
hmm, top level identifiers aren't meant to start with lowercase letters, maybe just inline this in Recipe as:

`name:(whiteSpace (upperIdent / lowerIdent))`

and/or a todo and we'll figure out how to clean this up (probably there aren't many recipes with names now).

Reply:
Naming a recipe seems fine. We might eventually want both?

Reply:
Done.